src/pages/products/index.js

```diff
@@ -45,7 +44,7 @@ export default function ProductsPage() {
         yellowButtonText1="All data submissions adhere to user-defined standards and structure"
         yellowButtonText2="Automated global identifiers tracks metadata across geographically distributed Song repositories"
         yellowButtonText3="Control the publication status of data with configurable data states"
-        isGrey={false}
+        isGrey={true}
```

Comment:
in react, just havving the props means true
```suggestion
        isGrey
```